public/locales/en/status.json

```diff
@@ -12,7 +12,7 @@
   "description": "Use this service with your file number (ESRF) to check the status of your passport application.",
   "esrf": {
     "error": {
-      "length": "The file number must be 8 characters long.",
+      "length": "The file number can be a maximum of 8 characters long.",
```

Comment:
I think this better:

`The file number can have a maximum of 8 alphanumeric characters.`

Reply:
I'm okay with this given the recent introduction of letters into the ESRF, but given the original `length` string was provided to us by the BAs this might change.